### DISCUSSION TITLE
my ticket vanished

### DISCUSSION THREAD

**Discussion by JoachimHenze** [Alex313031/Thorium-Win#218] [Q&A] [answered]
I opened two "Issues" for Thorium-legacy today 2024-May-11.
The first one "about extensions" was resolved as duplicate, and was therefore closed. I do understand that.

But the 2nd one regarding "Full context menu" is gone now entirely. I can see it neither in the "Closed issues section", nor was it moved to "Discussions" nor whatever. Where is it gone? Why can't I see it? What was the motivation/justification?

I think it is a bit unfriendly if stuff disappears into Nirvana. Closing with justification is fine. But now I am left in the dark as a user without knowing why.

**Comment by gz83** [comment, marked as answer]
https://github.com/Alex313031/thorium-legacy/issues/87

Your issue is here. Please note that Thorium-win is only applicable to win10 and above, while Thorium-legacy adds support for older versions of the system. Please choose the appropriate repo when filing an issue.
